pkg/analysis/optionalorrequired/testdata/src/a/a.go

```diff
@@ -32,6 +32,15 @@ type OptionalOrRequiredTestStruct struct {
 	// +required
 	RequiredMarker string
 
+    // +kubebuilder:validation:Optional
```

Comment:
Indentation is off

Reply:
Should be fixed in https://github.com/JoelSpeed/kal/pull/7/commits/8156a607d1efbaa3cb3702e4c5b7e712925d482f